app/middleware/error-handler.js

```diff
@@ -16,7 +16,7 @@ const {
   InvalidConfigurationError,
   ExpiredInviteError,
   RESTClientError,
-  GatewayTimeoutError
+  GatewayTimeoutError, GatewayTimeoutForAllServicesSearchError
```

Comment:
Optional: This could go on a new line to be consistent.

Reply:
done!